docker/poetry.Dockerfile

```diff
@@ -26,20 +20,13 @@ USER user
 WORKDIR /home/user/
 
 # Install poetry
+ENV PYTHONDONTWRITEBYTECODE 1 \
+    PYTHONUNBUFFERED 1
+
 RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
 ENV PATH="${PATH}:/home/user/.poetry/bin"
 
-CMD [ "bash" ]
-
-# If you prefer to clone and install traffic in the container (rather than bind
-# the volume, uncomment the following commands)
-
-# Clone and install traffic
-# RUN git config --global http.proxy ${PROXY}
-# RUN git clone https://github.com/xoolive/traffic
-
-# WORKDIR /home/user/traffic
-# RUN poetry install
+COPY pyproject.toml poetry.lock ./
 
-# By default, open a Python terminal
-# CMD [ "poetry", "run", "python" ]
\ No newline at end of file
+RUN poetry config virtualenvs.create false \
```

Comment:
If you set it false, I don't know where things go, to be honest...

Reply:
It's installed as the base env. Since we're in a dev container (of which the whole purpose it to use it just for that purpose), we don't want any additional environments besides that one.